### DISCUSSION TITLE
Unit Tests

### DISCUSSION THREAD

**Discussion by davidnmbond** [simple-odata-client/Simple.OData.Client#838] [General]
I suggest that all contributors get a full set of working Unit Tests.  Please confirm here.

**Comment by object** [comment]
I deleted WebApiTests that were using a Web site that no longer exists. The other tests should work.

**Comment by davidnmbond** [comment]
I have mirrored this on the modernization branch.
All (not skipped) unit tests now pass for me on both branches.
Other contributors please confirm that you also have passing unit tests.

Hopefully non-controversial rule: all unit tests must pass (with no deletions) from hereon in for any pull request to be accepted and any functional changes should come with full code coverage of the changes.

**Comment by davidnmbond** [comment]
I will look into automating code coverage metrics in the documentation.  Codacy is free for open source projects.

@object - please could you add me to the Simple.OData.Client Github organisation, so that I can enable the Codacy static code inspection metrics.
